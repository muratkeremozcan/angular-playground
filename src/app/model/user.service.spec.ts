import { UserService } from './user.service';

// this is a good example of isolated testing, a simple service that does not need any testing boilerplate
describe('UserService', () => {
  it('should create the service', () => {
    const user = new UserService();

    expect(user.isLoggedIn).toBe(true);
    expect(user.user.name).toBe('Sam Spade');
    expect(user).toMatchSnapshot();
  });
});
