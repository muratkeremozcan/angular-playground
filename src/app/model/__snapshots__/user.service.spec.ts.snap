// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserService should create the service 1`] = `
UserService {
  "isLoggedIn": true,
  "user": {
    "name": "Sam Spade",
  },
}
`;
